reference/audit-logs.mdx

```diff
@@ -0,0 +1,69 @@
+
+- Track changes made by your team to your observability posture.
+- Track monitoring performance.
+- Track billable usage.
+
+Audit logs are available to all users. Enterprise customers can query audit logs for the full time range. Other customers can query audit logs for the previous three days.
+
+## Explore audit logs
+
+1. Go to the **Explore** tab, and then click **Builder**.
+2. In the list of datasets, click **axiom-audit**.
+3. Optional: Customize your query to filter or summarize the audit logs. For more information, see [Explore data](/query-data/explore).
+4. Click **Run**.
+
+## Export audit logs
+
+1. Run the query to [display the audit logs](#explore-audit-logs).
+2. Click <img src="/doc-assets/icons/ellipsis-vertical.svg" className="inline-icon" alt="More icon" /> **More > Download as JSON**.
+
+## Restrict access to audit logs
+
+To restrict access to audit logs, use Axiom’s role-based access control to define who can access the `axiom-audit` dataset. For more information, see [Access](/reference/settings#access-overview).
+
+## List of trackable actions
+
+The `action` field specifies the type of activity that happened in your Axiom organization. Audit logs allow you to track the following actions:
+
+- **createAdvancedApiToken** means that a user created an advanced API token.
+- **createAnnotation** means that a user created an annotation.
+- **createBasicApiToken** means that a user created a basic API token.
+- **createDashboard** means that a user created a dashboard.
+- **createDataset** means that a user created a dataset.
+- **createMonitor** means that a user created a monitor.
+- **createNotifier** means that a user created a notifier.
+- **createOrg** means that a user created an organization.
+- **createPersonalAccessToken** means that a user created a personal access token.
+- **createUser** means that a user created another user.
+- **deleteDashboard** means that a user deleted a dashboard.
+- **deleteDataset** means that a user deleted a dataset.
+- **logout** means that a user logged out.
+- **notifierFailed** means that a notifier failed.
+- **notifierTriggered** means that a notifier triggered.
+- **removeUserFromOrg** means that a user removed another user from an organization.
+- **runAplQuery** means that a user ran an APL query.
+- **sendMessage** means that Axiom sent a message to a channel defined by a notifier.
+- **sendUserDeletedEmail** means that a user was deleted and Axiom sent out a notification email about the deletion.
+- **streamDataset** means that a user viewed the stream of a dataset.
+- **trimDataset** means that a user trimmed a dataset.
+- **updateDashboard** means that a user updated a dashboard.
+- **updateMonitor** means that a user deleted a monitor.
+- **updateUserSettings** means that a user updated their user settings.
+- **updateVirtualField** means that a user updated a virtual field.
```

Comment:
I worry that we'll struggle to keep up with these as more are added (indeed, this current list is not complete). I think the value of action should mostly be self-documenting?

Reply:
@benenharrington Thanks a lot for your review. I agree this is a maintenance hassle. I don't believe in self-documenting field names, but we can make the worst offenders easier to understand.

What do you think about renaming the following?
sendMessage --> sendMessageToNotificationChannel
sendUserDeletedEmail --> deleteUserSendNotificationEmail

Then we won't need this list, hopefully.

Reply:
@benenharrington Maybe we can include more context in the `message` field? Then we won't need this list. I'm happy to come up with a PR in the axiom repo if you think it's worthwhile.
Is there anything else I should sort out on this page?

Reply:
Happy with any changes to action names required to avoid documentation 👍 

No need to expand the message field, I think it would likely be quite a lot of work to have something meaningful here for all actions. Can iterate if we start to get questions

Reply:
> Is there anything else I should sort out on this page?

Nope!